client/src/domain/Community/CommunityListItem.js

```diff
@@ -0,0 +1,88 @@
+import "./CommunityListItem.css";
+import CommunityFallbackProfilePic from "../../assets/happy-students.jpg";
+import CommunitiesContext from "../../contexts/CommunitiesContext";
+
+import { useContext } from "react";
+import { useHistory } from "react-router";
+import { Link } from "react-router-dom";
+import { FaRegEdit, FaTrashAlt } from "react-icons/fa";
+import axios from "axios";
+
+// TODO: Get rid of that conditional wrapper again - shoudl be possible to just set an onLick-handler on the section-tag itself and basrta kanzla!
+const ConditionalWrapper = ({ condition, wrapper, children }) =>
+  condition ? wrapper(children) : children;
+
+export default function Community({ community, as }) {
+  console.log("yo community");
+  const renderAsListItem = as && as === "ListItem";
+  const communityProfilePicImgSrc =
+    community?.picture ?? CommunityFallbackProfilePic;
+
+  const { communities, setCommunities } = useContext(CommunitiesContext);
+
+  const history = useHistory();
+
+  const removeResource = (id) => {
+    axios
+      .delete(`api/communities/${id}`)
+      .then((res) => {
+        setCommunities(
+          communities.filter((resource) => {
+            return resource._id !== id;
+          })
+        );
+        // history.push("/");
+        history.goBack();
+      })
+      .catch((err) => {
+        console.log(
```

Comment:
console.logs won't be seen by the users, keep that in mind

Reply:
I planned on using a npm package for error- and crud-confirmation related user-notifications (the one is used for my Bello-App was quite nice) - so it's on my list already ...